tests/storage/exception_test.py

```diff
@@ -77,6 +84,25 @@ def test_info():
         }
 
 
+def test_VolumeGroupDoesNotExist():
+    # Require a VG name or UUID at initialization.
+    # Empty constructor shall raise.
+    with pytest.raises(ValueError):
+        e = storage_exception.VolumeGroupDoesNotExist()
+
+    # Expected error type is LVMCommandError.
+    with pytest.raises(TypeError):
+        e = storage_exception.VolumeGroupDoesNotExist("vg", error="error")
+
+    # Correct initialization, test info.
+    lvm_error = storage_exception.LVMCommandError(0, 1, 2, 3)
+    e = storage_exception.VolumeGroupDoesNotExist("vg", error=lvm_error)
+    assert e.info() == {
+        "code": e.code,
+        "message": str(e)
+    }
```

Comment:
I would test the formatted stringf instead of info(), so we can change the implementation later.

    e = storage_exception.VolumeGroupDoesNotExist("vg-name", error=lvm_error)
    assert e.error = fake_error

    formatted = str(e)
    assert "vg_name=vg-name" in formatted
    assert f"error={fake_error}" in formatted
    assert "vg_uuid=" not in formatted